src/taipy/config/_base_serializer.py

```diff
@@ -27,6 +27,16 @@
 from .unique_section import UniqueSection
 
 
+def _timedelta_to_str(obj: timedelta) -> str:
```

Comment:
This function should be a method of `_BaseSerializer` class right?

Reply:
Yes, I put it outside at first because it was also used on _CustomEncoder, but since the enconder is not needed as you guys pointed out, I have moved insed the class